### PR TITLE
Refactor: lite tests common

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,3 +4,6 @@
 [submodule "assets/shared"]
 	path = assets/shared
 	url = git@github.com:uktrade/lite-shared-assets.git
+[submodule "ui_automation_tests/shared"]
+	path = ui_automation_tests/shared
+	url = https://github.com/uktrade/lite-tests-common.git
```
Comment (markm5A8Z9F43913A) [inline .gitmodules]: We probably should keep git@github.com

**Comment by markm5A8Z9F43913A** [comment]
Apart from that one comment, I'm okay with approving this.

**Comment by markm5A8Z9F43913A** [comment]
Please read through this and rename the pull request: https://uktrade.atlassian.net/wiki/spaces/ILT/pages/830275975/GIT

File: .gitmodules (hunk 1 above)
Comment (BenAndrew-Kainos) [reply .gitmodules]: good point

**Comment by BenAndrew-Kainos** [comment]
> Please read through this and rename the pull request: https://uktrade.atlassian.net/wiki/spaces/ILT/pages/830275975/GIT

Cheers Mark. Will name it refactor:__ next time

**Pull request merged**